contracts/cairo/deploy.sh

```diff
@@ -8,19 +8,17 @@ COLOR_RESET='\033[0m'
 
 cd "$(dirname "$0")"
 
-load_env() {
-    unamestr=$(uname)
-    if [ "$unamestr" = 'Linux' ]; then
-      export $(sed '/^#/d; s/#.*$//' .env | xargs -d '\n')
-    elif [ "$unamestr" = 'FreeBSD' ] || [ "$unamestr" = 'Darwin' ]; then
-      export $(sed '/^#/d; s/#.*$//' .env | xargs -0)
-    fi
-}
-load_env
+if [ -f .env ]; then
+    echo "Sourcing cairo/.env file..."
+    source .env
+else
+    echo "Error: cairo/.env file not found!"
+    exit 1
+fi
 
 echo -e "${GREEN}\n=> [SN] Declare Escrow${COLOR_RESET}"
 ESCROW_CLASS_HASH=$(starkli declare \
-  --account $STARKNET_ACCOUNT --keystore $STARKNET_KEYSTORE \
+  --account $STARKNET_ACCOUNT --keystore $STARKNET_KEYSTORE --rpc $SN_RPC_URL \
```

Comment:
I think it might be best to remove --rpc param since its implementation is being handled in PR #51 . What do you think?

Reply:
I believe these changes should be in that PR precisely. Removing this would leave goerli as the default, which isn't a problem, but it limits the possibility of deploying on another network until that PR is merged.